test/e2e_node/system/types.go

```diff
@@ -114,7 +114,7 @@ var DefaultSysSpec = SysSpec{
 	Cgroups: []string{"cpu", "cpuacct", "cpuset", "devices", "freezer", "memory"},
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
-			Version: []string{`1\.(9|\d{2,})\..*`}, // Requires 1.9+
+			Version: []string{`1\.(9|\d?[0-2]{2,})\..*`}, // Requires 1.9+
```

Comment:
I'm not sure I understand the regexp here.

If we only want to officially support 1.9 - 1.12, could we change this to `1\.(9|1[0-2])\..*`?

@dchen1107 Two questions here:
1) Do we still support 1.9?
2) Are we going to say we don't support 1.13+ now?